development/image-builder/README.md

```diff
@@ -75,16 +114,21 @@ Usage of image-builder:
         Path to build directory context (default ".")
   -dockerfile string
         Path to Dockerfile file relative to context (default "Dockerfile")
+  -env-file string
+        Path to file with environment variables to be loaded in build
   -log-dir string
         Path to logs directory where GCB logs will be stored (default "/logs/artifacts")
   -name string
         Name of the image to be built
+  -platform value
+        Only supported with BuildKit. Platform of the image that is built
+  -repo string
+        Load repository-specific configuration, eg. signing configuration.
```

Comment:
```suggestion
        Load repository-specific configuration, for example, signing configuration
```

Why did this one line get a full stop at the end, and the other lines didn't? I'm removing it for consistency.

Reply:
Probably because that's how it's written in the code. It's just a copy-paste from `image-builder -h`